elementary/clients/s3/client.py

```diff
@@ -53,10 +54,66 @@ def send_report(
                     # We use report_filename because a path can not be an IndexDocument Suffix.
                     WebsiteConfiguration={"IndexDocument": {"Suffix": report_filename}},
                 )
+                bucket_website_url = self.get_bucket_website_url()
                 logger.info("Updated S3 bucket's website.")
         except botocore.exceptions.ClientError as ex:
             logger.exception("Failed to upload report to S3.")
             if self.tracking:
                 self.tracking.record_cli_internal_exception(ex)
-            return False
-        return True
+            return False, bucket_website_url
+        return True, bucket_website_url
+
+    def get_bucket_website_url(self) -> Optional[str]:
+        if self.config.update_bucket_website:
+            try:
+                bucket_name = self.config.s3_bucket_name
+                bucket_location = self.client.get_bucket_location(Bucket=bucket_name)[
+                    "LocationConstraint"
+                ]
+                aws_s3_website_url = self._get_aws_s3_website_url_from_location(
+                    bucket_location
+                )
+                return f"http://{bucket_name}.{aws_s3_website_url}"
+
+            except Exception as ex:
+                logger.warning(f"Unable to get bucket website URL: {ex}.")
+                return None
+        else:
+            return None
+
+    @staticmethod
+    def _get_aws_s3_website_url_from_location(location: str) -> str:
```

Comment:
You are right about the presigned URL, but this is how we guide our users for now because anyone with the signed link can access the presigned URL, so we decided not to do it for now.
We can talk about this, but this is our behaviour since day 1 and I don't think this is part of the task scope 🙂